<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserMgrPage</class>
 <widget class="QWidget" name="UserMgrPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>877</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>用户安全管理</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0" colspan="4">
       <widget class="QTableWidget" name="userTableWidget">
        <property name="dragEnabled">
         <bool>true</bool>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <column>
         <property name="text">
          <string>用户名</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>是否被锁定</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>UKey</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>解锁用户</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>更改UkeyPIN</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="pushButton_3">
        <property name="text">
         <string>绑定Ukey</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="pushButton_4">
        <property name="text">
         <string>解绑Ukey</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_15">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>250</height>
      </size>
     </property>
     <property name="title">
      <string>用户全局规则设置</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox_3">
        <property name="maximumSize">
         <size>
          <width>12222222</width>
          <height>65</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="title">
         <string>用户锁定设置</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="4">
            <widget class="QLineEdit" name="secLineEdit">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="0" column="7">
            <widget class="QPushButton" name="setTryLockButton">
             <property name="maximumSize">
              <size>
               <width>85</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>确定</string>
             </property>
            </widget>
           </item>
           <item row="0" column="6">
            <widget class="QComboBox" name="lockSvrComboBox">
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="toolTip">
              <string>法</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLineEdit" name="tmsLineEdit">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="toolTip">
              <string/>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="label_6">
             <property name="maximumSize">
              <size>
               <width>10000000</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="toolTip">
              <string>用户被锁定时登录失败的次数
默认值3</string>
             </property>
             <property name="text">
              <string>锁定次数:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="5">
            <widget class="QLabel" name="label_8">
             <property name="maximumSize">
              <size>
               <width>10000000</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="toolTip">
              <string>触发锁定的服务
默认值login</string>
             </property>
             <property name="text">
              <string>锁定服务:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QLabel" name="label_7">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="toolTip">
              <string>用户锁定到自动接锁的秒数
默认值30</string>
             </property>
             <property name="text">
              <string>锁定秒数:</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QGroupBox" name="groupBox_4">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>1677777</height>
         </size>
        </property>
        <property name="title">
         <string>密码规则设置</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_7">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="1" column="0">
            <widget class="QLabel" name="label">
             <property name="toolTip">
              <string>默认值：12</string>
             </property>
             <property name="text">
              <string>最短长度:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="9">
            <widget class="QLineEdit" name="othlenEdit"/>
           </item>
           <item row="1" column="6">
            <widget class="QLabel" name="label_4">
             <property name="toolTip">
              <string>默认值：1</string>
             </property>
             <property name="text">
              <string>小写字母个数:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="8">
            <widget class="QLabel" name="label_5">
             <property name="toolTip">
              <string>默认值：1</string>
             </property>
             <property name="text">
              <string>特殊字符个数:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="4">
            <widget class="QLabel" name="label_3">
             <property name="toolTip">
              <string>默认值：1</string>
             </property>
             <property name="text">
              <string>大写字母个数:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QLineEdit" name="dlenEdit"/>
           </item>
           <item row="1" column="7">
            <widget class="QLineEdit" name="lowlenEdit"/>
           </item>
           <item row="1" column="2">
            <widget class="QLabel" name="label_2">
             <property name="toolTip">
              <string>默认值：1</string>
             </property>
             <property name="text">
              <string>数字位数:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="5">
            <widget class="QLineEdit" name="uplenEdit"/>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="minlenEdit"/>
           </item>
           <item row="1" column="10">
            <widget class="QPushButton" name="setPwButton">
             <property name="text">
              <string>确定</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0" colspan="11">
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>规则说明：每增加一种密码类型，最小长度要求减一。例如最小长度为12时，增加小写字母限制，最小长度可以为11位。</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
